src/app/website/course-home-page/course-home-page.component.html

```diff
@@ -0,0 +1,9 @@
+<tf-weblayout [navBarType]="navBarMenuType">
+  <div Content>
+    <tf-caption [pageCaption]="page"></tf-caption>
+    <tf-popular-courses></tf-popular-courses>
+    <tf-why-learn></tf-why-learn>
+    <tf-testimonials [page]="testimonialType"></tf-testimonials>
+    <tf-our-job-partners [page]="ourJobPartner"></tf-our-job-partners>
+  </div>
```

Comment:
`tf-testimonials , tf-our-job-partners and tf-caption` should be part of layout and should change when the page changes. Because only the data model is changing and not the location or place of the component. Otherwise you are duplicating/repeating this for both pages now